cypress/e2e/main.cy.ts

```diff
@@ -9,6 +11,11 @@ describe('itch.io bundle claimer', () => {
       cy.task('log', `log in as ${$json.username}`)
     })
     cy.contains('button', 'Log in').click()
+    cy.readFile(secret_path).then(($json) => {
+      const { otp } = TOTP.generate($json.totp)
+      cy.get('[name=code]').type(otp)
+    })
+    cy.contains('button', 'Log in').click()
```

Comment:
I expect that this will fail for users who don't have MFA enabled. It's a simple fix, though. We just need to add some conditional logic to check if the page is asking for MFA or not. If MFA, then enter the OTP and click login. If not, then do nothing.

Here's an example of Cypress [conditional based on page text](https://docs.cypress.io/app/guides/conditional-testing#Conditionally-check-whether-an-element-has-certain-text).